L/LoweredCodeUtils/Compat.toml

```diff
@@ -1,9 +1,9 @@
-["0-0.1.0"]
+["0.1.0"]
 julia = "1.1-1"
 
 ["0.1.1-0.3.1"]
 JuliaInterpreter = "0"
 julia = "1"
 
-["0.3.2-0"]
+["0.3.2-0.3.3"]
```

Comment:
Why was this updated @StefanKarpinski?

Reply:
Boo. I know why. We decided it was easier to just copy `compress_versions` into Registrator but that means that the code that writes the compressed file is using the old `compress_versions` from `Pkg`. Ok, guess @nkottary or I need rebuild julia on that machine with a new Pkg version or just run Registrator with a newer Julia version.